rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/autoconfigure/RocketMQProperties.java

```diff
@@ -207,4 +220,32 @@ public void setCustomizedTraceTopic(String customizedTraceTopic) {
             this.customizedTraceTopic = customizedTraceTopic;
         }
     }
+
+    public Consumer getConsumer() {
+        return consumer;
+    }
+
+    public void setConsumer(Consumer consumer) {
+        this.consumer = consumer;
+    }
+
+    public static final class Consumer {
+        /**
+         * listener configuration container
+         * the pattern is like this:
+         * group1.topic1 = false
+         * group2.topic2 = true
+         * group3.topic3 = false
+         */
+        private Map<String, Map<String, Boolean>> listeners = new HashMap<>();
```

Comment:
I could not figure out the comment here. could you make a clear why you use the complicated map structure?

BTW, if I got a reasonable answer from your followup,  I really like to help to merge the pr.

Reply:
This PR is aimed to  adding  `rocketmq.consumer.listeners.<consumer-group-name>.<topic-name>.enabled = false` configuration for `@RocketMQMessageListener` to control it's initilization when container boostraped. So as least two dimensions (consumer-group-name and topic) should be illustrated, as a result, I use a two layered map to implement it.